### ISSUE TITLE
Possible without Genres?

### ISSUE THREAD

**Issue by mcp99999** [DevXen/MediaNamingScripts#2]
Thanks for the perfect musicbrainz-script!!!
Is it possible to set the folderstructure without the genres, so that the artist is on first?

**Comment by DevXen** [comment]
Yes. You will need to remove all the references to %grouping%  and all the references to %genre%

If it's an $if() statement you can remove the full statement. But you'll probably want to remove all of them in the script. I'd experiment to find what works for you.

%grouping% is the main genre.(Rock/Metal/country/etc.)

%genre% is the subgenre.  (Thrash Metal/Power Metal/Death Metal/etc)
